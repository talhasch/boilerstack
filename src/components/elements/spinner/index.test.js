import React from 'react';
import Spinner from './index';
import renderer from 'react-test-renderer';


it('render component', () => {
  const component = <Spinner/>;

  const tree = renderer
    .create(component)
    .toJSON();

  expect(tree).toMatchSnapshot();
});