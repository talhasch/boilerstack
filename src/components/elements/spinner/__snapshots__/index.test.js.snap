// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render component 1`] = `
<div
  className="spinner"
>
  <div
    className="double-bounce1"
  />
  <div
    className="double-bounce2"
  />
</div>
`;
